opentracing/src/main/java/opentracing/Span.java

```diff
@@ -25,16 +25,7 @@
   /**
    * Suitable for serializing over the wire, etc.
    */
-  TraceContext traceContext();
-
-  /**
-   * Denotes the beginning of a subordinate unit of work.
-   *
-   * @param operationName name of the operation represened by the new span from the perspective of
-   * the current service.
-   * @return a new child Span in "started" state.
-   */
-  Span startChild(String operationName);
+  SpanContext childContext();
```

Comment:
> what is the use case where one needs a child context, but does not need a child span?

That's not the change here, if I understand you correctly.

The method `Span startChild(String operationName);` was removed, because it's a convenience API method.

The `TraceContext traceContext();` method renamed to `SpanContext childContext();` so to avoid confusion as to whether this is to obtain the context for this object or the context for its children.